src/main/java/mate/academy/spring/validation/PasswordValidator.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.spring.validation;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import mate.academy.spring.dto.request.UserRequestDto;
+
+public class PasswordValidator implements ConstraintValidator<ValidPassword, UserRequestDto> {
+    @Override
+    public boolean isValid(UserRequestDto userRequestDto,
+                           ConstraintValidatorContext constraintValidatorContext) {
+        if (userRequestDto.getPassword() == null) {
+            return false;
+        }
+        if (userRequestDto.getRepeatPassword() == null) {
+            return false;
+        }
+        if (userRequestDto.getPassword().length() < 4) {
+            return false;
+        }
+        return userRequestDto.getRepeatPassword().equals(userRequestDto.getPassword());
```

Comment:
```suggestion
        return userRequestDto.getPassword() != null 
                        && userRequestDto.getPassword().length >= 4
                        && userRequestDto.getRepeatPassword().equals(userRequestDto.getPassword())
```

same, we can make it simpler